org.eclipse.transformer.cli/src/main/java/org/eclipse/transformer/cli/TransformerCLI.java

```diff
@@ -370,6 +370,11 @@ public ResultCode run() {
 			return logResult(ResultCode.ARGS_ERROR_RC);
 		}
 
+		if (getParsedArgs().getArgs().length >= 3) {
+			getLogger().error(consoleMarker, "Number of arguments must be less than 3");
```

Comment:
I assume you mean must NOT be less than 3.

Also this test seems to short circuit later tests which provide help: `getArgs().length == 0` below.

Reply:
No, I mean must be less than 3. 
Since only input and output can be specified as arguments, it will never be greater than 2.